project/LibraryManifestGenerator.scala

```diff
@@ -64,20 +70,46 @@ object LibraryManifestGenerator {
       canonicalPath.toString
     )
 
+    val allEnv = Map(
+      "ENSO_EDITION_PATH" -> file("distribution/editions").getCanonicalPath
+    ) ++ env
     val commandText = command.mkString(" ")
     log.debug(s"Running [$commandText].")
-    val exitCode = sys.process
-      .Process(
-        command,
-        cwd = Some(canonicalPath.getParentFile),
-        "ENSO_EDITION_PATH" -> file("distribution/editions").getCanonicalPath
-      )
-      .!
+    val procBldr = new java.lang.ProcessBuilder(asJava(command))
+    procBldr.directory(canonicalPath.getParentFile)
+    allEnv.foreach { case (key, value) =>
+      procBldr.environment().put(key, value)
+    }
+
+    val processOutLines: ListBuffer[String] = ListBuffer()
+    val captureOut = ProcessLogger(
+      fout = (s: String) => {
+        processOutLines.append("[stdout] " + s)
+      },
+      ferr = (s: String) => {
+        processOutLines.append("[stderr] " + s)
+      }
+    )
+    val exitCode = sys.process.Process(procBldr).!(captureOut)
     if (exitCode != 0) {
-      val message = s"Command [$commandText] has failed with code $exitCode."
+      val message = s"Command [$commandText] has failed with code $exitCode:"
       log.error(message)
+      log.error(processOutLines.mkString("\n"))
       throw new RuntimeException(message)
     }
   }
 
+  private def javaExecutable(): String = {
+    val jHome = System.getProperty("java.home")
+    if (jHome != null) {
+      if (Platform.isWindows) {
+        jHome + File.separator + "bin" + File.separator + "java.exe"
```

Comment:
- Scala has `/` operator...
- there already is some support for `OS` executable name
- https://github.com/enso-org/enso/blob/f3ea4db1965cb71f243d1e30d5f03c689cd50a73/project/DistributionPackage.scala#L649
- consider reuse